### PR TITLE
https://issues.redhat.com/browse/ACM-9084 Gatekeeper config update

### PR DESCRIPTION
**Pull request by dockerymick**
https://issues.redhat.com/browse/ACM-9084

Questions for @yiraeChristineKim:

Are users required to list all of the operations and should there be a DELETE operation listed?

### CODE REVIEW

File: governance/create_gatekeeper.adoc [hunk 1]
```diff
@@ -93,6 +98,7 @@ spec:
     some-annotation: "this is a test"
     other-annotation: "another test"
 ----
+//are users required to add all of these operations? Should there be a DELETE operation?
```
Comment (yiraeChristineKim) [inline governance/create_gatekeeper.adoc]: Oh is this for me? nono it just example of operations, it doesn't matter we add `DELETE` or not

Comment (yiraeChristineKim) [reply governance/create_gatekeeper.adoc]: Oh, also we didn't separate the gatekeeper version yet. This feature is only available in v3.14

**Review by yiraeChristineKim — APPROVED** [review]

File: governance/create_gatekeeper.adoc (hunk 1 above)
Comment (yiraeChristineKim) [reply governance/create_gatekeeper.adoc]: Also we add in` oc explain` so should be fine

Comment (dockerymick) [reply governance/create_gatekeeper.adoc]: If this is not available, it shouldn't be documented. closing this PR

File: governance/create_gatekeeper.adoc [hunk 2]
```diff
@@ -93,6 +98,8 @@ spec:
     some-annotation: "this is a test"
     other-annotation: "another test"
 ----
++
+You can use the following values to manage your webhook operations. Use the following calues for the `operations` parameter, `"CREATE"`, `"UPDATE"`, `"CONNECT"`, and `"DELETE"`.
```
Comment (swopebe) [inline governance/create_gatekeeper.adoc]:
```suggestion
You can use the following values to manage your webhook operations. Use the following values for the `operations` parameter, `"CREATE"`, `"UPDATE"`, `"CONNECT"`, and `"DELETE"`.
```

File: release_notes/whats_new.adoc [hunk 3]
```diff
@@ -50,6 +50,10 @@ See link:../observability/observe_environments_intro.adoc#observing-environments
 [#governance-whats-new]
 == Governance
 
+* Configure the operations of the Gatekeeper operator validating webhook to manage admission events. See link:../governance/create_gatekeeper.adoc#managing-gatekeeper-operator-policies[Managing Gatekeeper operator policies] for details.
```
Comment (swopebe) [inline release_notes/whats_new.adoc]: confused about this:

of the Gatekeeper operator validating webhook

Is that a whole thing or should there be a "by" in there?

**Review by swopebe — APPROVED** [review]
Two small comments.

File: release_notes/whats_new.adoc (hunk 3 above)
Comment (dockerymick) [reply release_notes/whats_new.adoc]: "Gatekeeper operator validating webhook" is one word. @yiraeChristineKim does **Gatekeeper operator webhook** work here as well? Is it accurate to refer to the webhook like this? Also pulling in @dhaiducek

**Comment by dockerymick** [comment]
@swopebe thanks for your review, suggestions, and approval!

File: release_notes/whats_new.adoc (hunk 3 above)
Comment (yiraeChristineKim) [reply release_notes/whats_new.adoc]: The operation verbs apply to Gatekeeper operator validating webhook(exactly gatekeeper's validatingwebhookconfiguration)

Comment (yiraeChristineKim) [reply release_notes/whats_new.adoc]: It could be confusing... it is  gatekeeper's validatingwebhookconfiguration not Gatekeeper operator's

Comment (dockerymick) [reply release_notes/whats_new.adoc]: Thanks for confirming!

Comment (dockerymick) [inline release_notes/whats_new.adoc]:
```suggestion
* Configure the operations of the Gatekeeper operator webhook to manage admission events. See link:../governance/create_gatekeeper.adoc#managing-gatekeeper-operator-policies[Managing Gatekeeper operator policies] for details.
```

**Comment by dockerymick** [comment]
@yiraeChristineKim @swopebe thanks for the reviews and approvals! Merging this PR

**Pull request merged**